File: ImageButton/app/src/main/java/com/example/imagebutton/MainActivity.java
package com.example.imagebutton;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void boton1(View view){
        Toast.makeText(this, "La geekipedia de Ernesto", Toast.LENGTH_SHORT).show();
    }

    public void boton2(View view){
        Toast.makeText(this, "Esta es una mano", Toast.LENGTH_SHORT).show();
    }
}

File: ExampleMVVM/app/src/main/java/com/escm/examplemvvm/MainActivity.kt
package com.escm.examplemvvm

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import com.escm.examplemvvm.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //setContentView(R.layout.activity_main)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnToast.setOnClickListener{
            toast()
        }
    }

    private fun toast(){
        Toast.makeText(this,"Mensaje!",Toast.LENGTH_LONG).show()
    }
}